wiki/Difficulties/osu!/Easy/en.md

```diff
@@ -12,20 +14,20 @@ And try to avoid these:
 
 -   stacked notes
 -   overlapping notes
--   [Short Repeating Sliders](/wiki/Mapping_Techniques/Slider/Short_Repeating_Sliders/)
+-   [Short repeating sliders](/wiki/Mapping_Techniques/Slider/Short_Repeating_Sliders/)
```

Comment:
the link's location doesn't exist (because stuff I did); just remove the link for now